lib/cc/analyzer/engine.rb

```diff
@@ -16,7 +16,9 @@ def initialize(name, metadata, code_path, config, label)
         @label = label.to_s
       end
 
-      def run(stdout_io, stderr_io = StringIO.new)
+      def run(stdout_io, stderr_io = StringIO.new, container_log = NullContainerLog.new)
+        container_log.started(@metadata["image"])
```

Comment:
Thoughts on creating an initializer which takes an image name? It just seems strange that only one of the log methods takes an image but they all relate to the same image.